### ISSUE TITLE
How to find allelels

### ISSUE THREAD

**Issue by Thomieh73** [ksahlin/NGSpeciesID#19]
Hi, I am trying to process amplicon sequences of a gene (daa) that in some animals has multiple allelels. Now I am wondering how to adjust the clustering parameters so that I can split the main cluster into two clusters.

This is my command so far: 
```
NGSpeciesID --ont --consensus\
 --sample_size 500 \
  --m 1000 --s 50
 --medaka \
--aligned_threshold 0.2\
 --fastq ${MYSAMPLE}.fastq\
  --outfolder ${MYSAMPLE}.${NUMBER}r.${SIZE}.out \
--primer_file ${PRIMERS}
```
The first thing that crossed my mind was to make the `aligned_threshold` stricter? But do you have experience with the other settings?
Which one would be good to use as well?

**Comment by ksahlin** [comment]
Hi @Thomieh73,
 
Multiple-allele separation is not yet implemented in NGSpeciesID. However, this is on the radar as soon as we have the right person to implement this.

It depends a bit on how similar the alleles are. If we are talking 99% identical, it may be difficult/impossible with NGSpeciesID.

In case ~97% or lower identity, you could try increasing the values of `--aligned_threshold`, `--mapped_threshold` and `--rc_identity_threshold`. A guesstimate is to set them  `--aligned_threshold 0.9 --mapped_threshold 0.9 --rc_identity_threshold X`, where X is a fraction a bit above the expected allele difference (so that in the case of multiple clusters, NGSpeciesID doesn't merge them thinking they are reverse complements).

**Comment by Thomieh73** [comment]
Thanks for the feedback @ksahlin, I will try that. I had already figured out that the `--rc_identity_threshold` was key here, since I noticed that my alleles could be grouped together when they are too close :-)

**Comment by ksahlin** [comment]
Yes, it seems a bit shaky. A multiple-allele separation feature is definitely desirable in these cases. Hope to add such a feature soon.

**Comment by Thomieh73** [comment]
That would be great. But for now, I will close this issue.  Thanks for making a nice tool, I do trust it for the purpose you specified, and I recommended it to my fellow colleagues working with meta-barcoding data.
